astroid/nodes/scoped_nodes.py

```diff
@@ -1606,6 +1608,7 @@ def blockstart_tolineno(self):
 
         :type: int
         """
+        # pylint: disable-next=no-member
```

Comment:
I'm a bit hesitant about these `pylint: disable` statements. Maybe it would be better to remove the annotations in `astroid/exceptions.py` for now.
They don't add much as the error is only used in two locations anyway.

Reply:
I'm going to fix this in another PR. I'll tag you!

Reply:
See https://github.com/PyCQA/astroid/pull/1174

Reply:
I see now! These are actual bugs, only `Arguments` should be assigned to `self.args` not a `list`.
I was under the impression that these are `pylint` bugs.

In that case, these are ok.